.github/actions/test/action.yml

```diff
@@ -12,7 +17,7 @@ runs:
       working-directory: voluncheer
       run: |
         coverage run --source "." manage.py test
-        coverage lcov --fail-under=${MINIMUM_COVERAGE}
+        coverage lcov --fail-under=${{ inputs.minimum-coverage }}
```

Comment:
Should this be INPUTS_MINIMUM-COVERAGE? See: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs

Reply:
Maybe both are supported?

This is the reference I was using with regards to these contexts: https://docs.github.com/en/actions/learn-github-actions/contexts#inputs-context